xn2v/hetnode2vec.py

```diff
@@ -109,8 +109,7 @@ def simulate_walks(self, num_walks: int, walk_length: int):
             random.shuffle(nodes)
             for node in nodes:
                 walks.append(self.node2vec_walk(walk_length=walk_length, start_node=node))
-
-        return walks
+        return tf.ragged.constant(walks)
```

Comment:
Are we assuming that a ragged tensor will be used for all input types (i.e. list of lists of same length and list of lists of differing lengths)? If so, do we know that this tf object works for both data types?

Reply:
I assumed that it does not matter if the size is the same, and I could see no disadvantage of using a RaggedTensor. Is there a better option?

Reply:
I think that's probably fine (assuming we know it can be used regardless of the input data type), if that is the case then I think we can also update the scripts to remove the lines that check the structure of the input data before outputting the `tf` object (example is what happens at the end of the `text_encoder_tf.py` script.